fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1233,11 +1233,11 @@ executeAsStatement
     ;
 
 createRoleStatement
-    : CREATE ROLE (IF NOT EXISTS)? identifierOrString
+    : CREATE ROLE (IF NOT EXISTS)? roleList
```

Comment:
why we need to create multi roles on time?

Reply:
This is not a bugfix, just a new mini feature. We don't have to support it, but we can provide this ability.